src/Swashbuckle.SwaggerGen/Annotations/SwaggerAttributesOperationFilter.cs

```diff
@@ -2,6 +2,7 @@
 using System.Linq;
 using System.Net;
 using Swashbuckle.SwaggerGen.Generator;
+using Swashbuckle.SwaggerGen.Extensions;
```

Comment:
The extensions (ApiDescriptionExtensions, JsonContractExtensions etc.) are specific to the "Generator" functionality and so I would like to keep them in the same namespace. It's just a styling preference - I prefer to namespace by functionality rather than technical groupings.